pyproject.toml

```diff
@@ -81,6 +81,10 @@ profile = "black"
 src_paths = ["src"]
 
 
+[tool.pyright]
+include = ["src"]
```

Comment:
Should `tests` also be checked?

Reply:
Ah, good catch! I missed that we have `tests` in two places :) 

Fix inbound